### PR TITLE
docs: refine readme, update intro, highlight and images

### PR DESCRIPTION
**Pull request by elliotxx**
## What type of PR is this?

/kind documentation

## What this PR does / why we need it:

Refine readme again. Update the introduction, highlight and images.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,52 +28,54 @@
 
 ## What is Karpor?
 
-Karpor is a Modern **Kubernetes Explorer** focusing on **🔍 Search**, **💡 Insight** and **🤖 Intelligence**. It has keywords such as `self-host`, `non-invasive`, `read-only`, `secure compliance`, `multi-cloud/multi-cluster support`, `custom logical view`, and can serve as a **Kubernetes Data Plane** to reduce the cost of discovering and understanding kubernetes resources.
+Karpor is Intelligence for Kubernetes. It brings advanced **🔍 Search**, **💡 Insight** and **✨ AI** to Kubernetes. With Karpor, you can gain more visibility into your Kubernetes clusters across any clouds and empower developers to ship faster.
```
Comment (adohe) [inline README.md:31]: gain more visibility -> gain crucial visibility

File: README.md [hunk 2]
```diff
@@ -29,52 +28,54 @@
...
+<h3 align="center">🔍 Search</h3>
 
 <table>
   <tr>
-    <td style="vertical-align: middle;">
-      <b>⚡️ Lightweight and Easy to Setup</b><br />One `helm` is done.<br /><br/>
-      <b>📦 Out of the Box</b><br />Built-in security and compliance policies, resource sync strategy, resource transform rule, and topology relationship definitions.<br /><br />
-      <b>🔍 Quickly search and locate resource(s)</b><br />Quickly search and locate resource(s) of interest across a large number of clusters in a user-friendly way.
+    <td>
+      <b>⚡️ Automatic Syncing</b><br />Automatically synchronize your resources across any clusters managed by the multi-cloud platform.<br /><br/>
```
Comment (adohe) [inline README.md:44]: The aspect ratio of the text and image is not very harmonious.

File: README.md (hunk 1 above)
Comment (elliotxx) [reply README.md]: Resolved

File: README.md (hunk 2 above)
Comment (elliotxx) [reply README.md:44]: Yes, I also found this problem, but in GitHub Markdown, it does not support CSS syntax such as style, width ... (Set up and not work)

**Review by adohe — APPROVED** [review]
/lgtm

**Pull request merged**